conference.js

```diff
@@ -1359,14 +1359,17 @@ export default {
 
     /**
      * Toggles between screensharing and camera video.
+     *
+     * @param {boolean} toggle - Indicates whether screen sharing should be
```

Comment:
I have tested to call it 2 times with true because this is the only use case that we have. I don't see a reason why 2 times false won't work. But I would suggest to not waste time on this since we currently don't have this use case and I'm not sure if we are going to have it ever. I'll try to add more info to the docs.